packages/@dcl/sdk/cli/utils/fs.ts

```diff
@@ -25,12 +25,28 @@ export async function ensureFolder(path: string | string[]): Promise<void> {
   await _this.ensureFolder(path.slice(1))
 }
 
+/**
+ * Read's a directory files
+ * @param dir Directory to read files from
+ */
+export async function readdir(dir: string): Promise<string[]> {
+  return fs.readdir(dir)
+}
+
+/**
+ * Read's a file contents
+ * @param path File to read content from
+ */
+export async function readFile(path: string): Promise<string> {
+  return fs.readFile(path, 'utf8')
+}
+
 /**
  * Check's if directory is empty
  * @param dir Directory to check for emptyness
  */
 export async function isDirectoryEmpty(dir: string): Promise<boolean> {
-  const files = await fs.readdir(dir)
+  const files = await _this.readdir(dir)
```

Comment:
cant we call `readdir` function istead of `_this.` .
```suggestion
  const files = await readdir(dir)
```

Reply:
not if we want jest to be able to mock it 😛 

https://stackoverflow.com/questions/45111198/how-to-mock-functions-in-the-same-module-using-jest